scripts/shared/nodeinfo/nodeinfo-update.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+cp /opt/tomesh/nodeinfo.json /tmp
+
+# Replace placeholders with dynamic info
+sed -i -e "s/__REPO__/$(git remote get-url origin | awk -F / '{print $5}'| cut -d '.' -f1)/g" /tmp/nodeinfo.json
+sed -i -e "s/__BRANCH__/$(git rev-parse --abbrev-ref HEAD)/g" /tmp/nodeinfo.json
+sed -i -e "s/__COMMIT__/$(git rev-parse HEAD)/g" /tmp/nodeinfo.json
+sed -i -e "s/__INSTALLED__/$(date)/g" /tmp/nodeinfo.json
+
+sed -i -e "s/__KEY__/$(grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g')/g" /tmp/nodeinfo.json
+sed -i -e "s/__ORG__/$MESH_NAME/g" /tmp/nodeinfo.json
+
+services=$(run-parts /opt/tomesh/nodeinfo.d/ | sed ':a $!{N; ba}; s/\n/\\n/g')
+sed -i -e "s/__SERVICES__/$services/g" /tmp/nodeinfo.json
```

Comment:
Can we `jq` instead of `sed` and not use macros here?

Reply:
For this pr followed make world lead 
Question is  favour jq over sed everywhere or less dependencies

Also jq does not work with cjdouter.conf because of comments

Reply:
@darkdrgn2k @benhylau I feel like since jq is already a dependency it should be used where it's possible. If it doesn't work with cjdroute.conf that sucks, but when it can be used, it should, because it's much easier to write and read.

Reply:
Just a note - JQ is actually NOT a dependency except for this particular instance.

It was removed in other instance because it didn't work and would not work.